src/DegenGambit.sol

```diff
@@ -605,6 +605,21 @@ contract DegenGambit is ERC20, ReentrancyGuard {
         }
     }
 
+    // Payout Estimate function to easily display current payouts estimate at time of function call
+    function payoutEstimate(
```

Comment:
I want to see a test or multiple tests which test the values of this `view` against those of the `payout` method.

Actually, in the integration guide we should perhaps just tell users to use the `payout` method?

Or maybe we should just return something like:

```solidity
return (payout(0,0,0), payout(1,1,1), payout(15, 16, 15), payout(15, 16, 17), payout(17, 17, 17));
```

(My indices may be incorrect, but the idea stands).